Parser/action_helpers.c

```diff
@@ -1693,4 +1693,4 @@ _PyPegen_concatenate_strings(Parser *p, asdl_expr_seq *strings,
 
     assert(current_pos == n_elements);
     return _PyAST_JoinedStr(values, lineno, col_offset, end_lineno, end_col_offset, p->arena);
-}
+}
```

Comment:
revert this.

Reply:
Will do! In any case don't worry too much for now as we are just in the exploratory phase. Once we are happy I will clean everything